mineral-ui/mineral-ui
# [Avatar][Card] New components

### Description
- Add Avatar
- Add CardStatus
- Add DangerSimple, SuccessSimple, and WarningSimple to bundled Icons
- Add/update/remove demo examples
- Add/update/remove snapshots
- Replace avatar placeholder image with a better one
- Add getColor & getReadableTextColor utilities to the colors directory
    - Quantify the readable color for each color in our palette

### Motivation and context

Closes #491 

### Screenshots, videos, or demo, if appropriate


https://491-card-supplementals--mineral-ui.netlify.com/components/avatar
https://491-card-supplementals--mineral-ui.netlify.com/components/card-status

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to Avatar and CardStatus
3. Confirm things look right

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist




### How does this PR make you feel?

![Avatar](https://media.giphy.com/media/ZvJ0bHvAy1N9S/giphy.gif)
